src/database.py

```diff
@@ -0,0 +1,14 @@
+"""Periferals required for SQLAlchemy to function."""
+
+from sqlalchemy import create_engine
+from sqlalchemy.orm import declarative_base, scoped_session, sessionmaker
+
+# will allow us to connect to PostgreSQL database
+engine = create_engine("postgresql://postgres:@localhost:5432/postgres")
```

Comment:
Let's make connection string to be configurable.

What we want here is to have an ability to provide different connection strings to our application without rewriting code. Why do we need this? Because our application may run on different databases depending on environment, for example:
- for dev environment we need `postgresql://postgres:@localhost:5432/postgres`
- for test environment we will need `postgresql://testuser:somepassword@test-stand-url.com:5432/testdb`
- for prod environment we will need `postgresql://superuser:supersecretpassword@prod-stand-url:5432/forum123_db`

So we need an ability to provide different connection strings in different environments.